react-firebase-chat/src/App.js

```diff
@@ -10,15 +10,37 @@ import { useAuthState } from 'react-firebase-hooks/auth';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 firebase.initializeApp({
-  apiKey: "AIzaSyASMgiTftDVoNsFFzlwIuhEOgyHSX5QJq8",
-  authDomain: "project-2-dbfa6.firebaseapp.com",
-  projectId: "project-2-dbfa6",
-  storageBucket: "project-2-dbfa6.appspot.com",
-  messagingSenderId: "496736545958",
-  appId: "1:496736545958:web:4355c8b0cb8406e9aa5867",
-  measurementId: "G-JKF7VKGKEX"
```

Comment:
Firebase App Configuration has been removed and replaced with a new configuration.